src/foam/nanos/u2/navigation/NavigationView.js

```diff
@@ -0,0 +1,31 @@
+foam.CLASS({
+  package: 'foam.nanos.u2.navigation',
+  name: 'NavigationView',
+  extends: 'foam.u2.View',
+   documentation: 'Navigation bars',
+   css: `
+    ^ .top-nav {
```

Comment:
Update your CSS to `^top-nav` now, not `^ .top-nav` as we're using myClass('top-nav')